util/ConfigFileProcess.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _ConfigFileProcess_h_
+#define _ConfigFileProcess_h_
+
+#include <vector>
+#include <string>
+#include <map>
```

Comment:
Okay, I was still operating from the old model where all includes are in the header. But I've read some blogs on the topic, so I understand now what I should be aiming for.